exercises/02-generate-odata-clients.md

```diff
@@ -0,0 +1,34 @@
+# Generate your own OData clients
```

Comment:
do we really need this section when we're allowed to publish generated clients?

Reply:
I think since this meant for the reader to follow as a step-by-step exercise, running the generation process is also part of it. 

But then we should also not check-in the generated clients, right? 
Also in the ticket, download and generate are separate steps for both services (the latter with an additional option)
- download s4 service spec + generate s4 client
- download sfsf service spec + generate sfsf client with --clearOutputDir

But here, we are doing them in one go.  cc @marikaner